artichoke-backend/src/extn/core/time/mruby.rs

```diff
@@ -19,7 +19,7 @@ pub fn init(interp: &mut Artichoke) -> InitializeResult<()> {
         .value_is_rust_object()
         // Constructor
         .add_self_method("now", time_self_now, sys::mrb_args_none())?
-        .add_self_method("at", time_self_at, sys::mrb_args_req(1))?
+        .add_self_method("at", time_self_at, sys::mrb_args_req(1) | sys::mrb_args_opt(3))?
```

Comment:
My understanding is this should work. However it funnily enough seems to not do anything regarding the validation. Is it expected to, or is this just for documentation to some degree?

Reply:
mruby only raises an ArgumentError from the VM when there is an arg mismatch for methods that take no args.

It is the call to the `mrb_get_args!` that triggers arity checking.

See:

- https://github.com/mruby/mruby/issues/4688

Reply:
Hrrrmmm, I must have been thinking about something else when I wrote this comment, my bad! I realized just now that it's working as expected :+1: